src/oemof_tabular_plugins/wefe/facades/functions.py

```diff
@@ -307,7 +308,10 @@ def vap_pressure(t):
     e: numeric
         saturation vapour pressure [kPa]
     """
-    e = 0.6108 * np.exp(17.27 * t / (t + 237.3))
+    c1 = 0.6108 # [kPa]
+    c2 = 17.27  # [-]
+    c3 = 237.3  # [K]
```

Comment:
for `c3` something like `T_<some informative name>` would have been better to directly carry the meaning that this is a temperature

Reply:
for `c_1` something like `P_<some informative name>`